.github/workflows/update-helm-chart.yml

```diff
@@ -0,0 +1,138 @@
+name: Upgrade Helm Chart
+on:
+  schedule:
+    - cron: "0 1 * * 1-5"
```

Comment:
let's try to run it as part of PR just to test it for now (right before merging we will adapt that back)